frontend/src/pages/my-projects/components/create-project-form/create-project-form.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Input } from '~/libs/components/components.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useAppForm, useAppSelector, useCallback } from '~/libs/hooks/hooks.js';
+import {
+  type ProjectCreateRequestDto,
+  projectCreateValidationSchema,
+} from '~/packages/projects/projects.js';
+import { type UserAuthResponse } from '~/packages/users/users.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  onSubmit: (payload: ProjectCreateRequestDto) => void;
+  className?: string | undefined;
```

Comment:
Why `string | undefined`?

Reply:
Due to this rule - https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-simpleland/blob/development/tsconfig.json#L6

take a look at the documentation